### DISCUSSION TITLE
Translating Scratch Addons To Turkish

### DISCUSSION THREAD

**Discussion by Thomas282-coder** [ScratchAddons/ScratchAddons#1053] [Translation] [answered]
Hello, my Transifex username is **Translator20**, my Scratch username is **Thomas282**, and I want to translate it into Turkish.

**Comment by GrahamSH-LLK** [comment, marked as answer]
Okay, I've added you as a translator to Turkish on Transifex.

**Reply by Thomas282-coder** [reply]
Thank you 👍
